builds.json

```diff
@@ -1,6 +1,38 @@
 {
   "builds" :
   [
+    {
+      "version": "7.0.0",
```

Comment:
Just as a side note. Once CVA is adapted to these latest models, we don't need build version 7.0.0. We can stick with 6.0.0